py-rse/packaging.Rmd

```diff
@@ -647,52 +624,55 @@ warning: check: missing required meta-data: url
 
 warning: check: missing meta-data: if 'author' supplied, 'author_email' must be supplied too
 
-creating zipf-0.1
-creating zipf-0.1/zipf
-creating zipf-0.1/zipf.egg-info
-copying files to zipf-0.1...
-copying README.md -> zipf-0.1
-copying setup.py -> zipf-0.1
-copying zipf/collate.py -> zipf-0.1/zipf
-copying zipf/countwords.py -> zipf-0.1/zipf
-copying zipf/plotcounts.py -> zipf-0.1/zipf
-copying zipf/utilities.py -> zipf-0.1/zipf
-copying zipf.egg-info/PKG-INFO -> zipf-0.1/zipf.egg-info
-copying zipf.egg-info/SOURCES.txt -> zipf-0.1/zipf.egg-info
-copying zipf.egg-info/dependency_links.txt -> zipf-0.1/zipf.egg-info
-copying zipf.egg-info/entry_points.txt -> zipf-0.1/zipf.egg-info
-copying zipf.egg-info/requires.txt -> zipf-0.1/zipf.egg-info
-copying zipf.egg-info/top_level.txt -> zipf-0.1/zipf.egg-info
-Writing zipf-0.1/setup.cfg
+creating zipf-0.1.0
+creating zipf-0.1.0/zipf
+creating zipf-0.1.0/zipf.egg-info
+copying files to zipf-0.1.0...
+copying README.md -> zipf-0.1.0
+copying setup.py -> zipf-0.1.0
+copying zipf/collate.py -> zipf-0.1.0/zipf
+copying zipf/countwords.py -> zipf-0.1.0/zipf
+copying zipf/plotcounts.py -> zipf-0.1.0/zipf
+copying zipf/utilities.py -> zipf-0.1.0/zipf
+copying zipf.egg-info/PKG-INFO -> zipf-0.1.0/zipf.egg-info
+copying zipf.egg-info/SOURCES.txt -> zipf-0.1.0/zipf.egg-info
+copying zipf.egg-info/dependency_links.txt -> zipf-0.1.0/zipf.egg-info
+copying zipf.egg-info/entry_points.txt -> zipf-0.1.0/zipf.egg-info
+copying zipf.egg-info/requires.txt -> zipf-0.1.0/zipf.egg-info
+copying zipf.egg-info/top_level.txt -> zipf-0.1.0/zipf.egg-info
+Writing zipf-0.1.0/setup.cfg
 creating dist
 Creating tar archive
-removing 'zipf-0.1' (and everything under it)
+removing 'zipf-0.1.0' (and everything under it)
 ```
 
-These distribution files can now be distributed via PyPI,
-the standard Python repository for Python packages.
-There is a TestPyPI repository that lets us test putting packages on PyPI
-without having them appear on the standard repository;
-let's start with that.
-Go to [test.pypi.org](https://test.pypi.org),
-create an account,
-and confirm your email.
+These distribution files can now be distributed via [PyPI][pypi],
+the standard repository for Python packages.
+Before doing that,
+though,
+we can put `zipf` on [TestPyPI][testpypi],
+which lets us test package distribution
```

Comment:
```suggestion
which lets us test package distribution files
```

Reply:
Not sure about the wording here, it was confusing to me... could you confirm this?